cli/args/config_file.rs

```diff
@@ -430,6 +430,12 @@ pub struct TestConfig {
   pub files: FilesConfig,
 }
 
+#[derive(Clone, Debug, Deserialize)]
+pub enum LockConfig {
+  Bool(bool),
+  String(String),
```

Comment:
I think the enum variant should be named what it is to be more clear: Path or Filename. I'm not sure if using the `Path` type is possible with deserialization. If it is, I would prefer it over using String.

```suggestion
  Filename(String),
```

Reply:
Would this be consistent?

Reply:
Can you now resolve this conversation, @EdJoPaTo?

Reply:
Nope, the button is not there. Looks like I have no permissions to do so?